sklearn/covariance/outlier_detection.py

```diff
@@ -28,27 +29,38 @@ class OutlierDetectionMixin(object):
         The amount of contamination of the data set, i.e. the proportion
         of outliers in the data set.
 
+    raw_decision : bool
+        Whether or not to consider raw (negative) Mahalanobis distances as the
+        decision function. Otherwise returns `[-(dist)**(1/3) + thres**(1/3)]`.
+        Must be False` (default) for compatibility with the others outlier
+        detection tools.
+
     Notes
     -----
     Outlier detection from covariance estimation may break or not
     perform well in high-dimensional settings. In particular, one will
     always take care to work with ``n_samples > n_features ** 2``.
 
+    The `raw_decision` kwarg now returns the negative Mahalanobis distance as a
```

Comment:
I am not sure this belongs in `Notes`. If nothing else, we'll forget to remove it when deprecation concludes.